### DISCUSSION TITLE
Right arrow to open article in browser

### DISCUSSION THREAD

**Discussion by briandfoy** [ViennaRSS/vienna-rss#1548] [Ideas]
I've been using Vienna for awhile, and NetNewswire for many, many years before then. Thanks for keeping this going! And I'd gladly sponsor this project if that option were available.

I typically don't read articles in Vienna, but quickly go through all my feeds and open the interesting articles in an external browser. Once I've gone through all the feeds, I then switch over to my browser. With NNW, I was able to use the right arrow to open the article in the external browser, which was convenient because I was already navigating with arrow keys to go through the left pane of folder and then the article lists. That part works fine in Vienna.

If a right arrow in the articles pane opened the article, that would be quite nice.

**Comment by barijaona** [comment]
We already have the Enter key doing this.

**Reply by TAKeanice** [reply]
I agree with @briandfoy, I also missed that option when I switched from NetNewsWire many years ago

**Reply by barijaona** [reply]
and Option-Enter in the articles pane will open the article in the alternate browser (in your case, it will open it in a Vienna tab).

**Reply by barijaona** [reply]
It seems that current versions of Netnewswire (I tested 5.0.3 and 6.1) do not support anymore the right arrow key for doing this, but the Enter key…

**Reply by briandfoy** [reply]
The Enter key is pretty far away from the arrow keys, which means I'd constantly have to move my hand rather to switch from the arrow keys to enter. Not a huge deal, but going through several hundred articles quickly is easier when I can keep my fingers on the arrow keys. I've also appreciated configurable action keys in other apps, though.

Not a big deal because I've been dealing with it, obviously. I don't really care what NetNewsWire does now because I stopped using them for a reason, and Vienna solves that reason. :)
